src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/util/ProcessUtil.java

```diff
@@ -76,6 +76,18 @@ public static CommandLine buildKillCommandLine(String pid)
         
         return command;
     }
+    
+    /**
+     * After forcibly killing the ES instance, remove the pid file.
+     * @param baseDir the base directory where the pid file is
+     */
+    public static void cleanupPid(String baseDir)
+    {
+        if (SystemUtils.IS_OS_WINDOWS)
```

Comment:
Wondering why you want the pid file delete to be done only on Win.

Reply:
That's because on *NIX `Process.destroy()` sends a SIGTERM, so ES gets the chance to do proper cleanup. On Win, a force-kill is done and ES cannot cleanup anything.

Reply:
I thought I made a comment about the `/f` flag on taskkill, but most probably it wasn't published it since I cannot find it.
Anyway, I was asking if we should rather remove the `/f` flag (which stands for force, similar to SIGKILL on linux) from taskkill; that would give the ES process the chance to shut down nicely. If so, then we don't need the pid file deletion (or it can stay, to catch the cases where the file doesn't get removed).